### PR TITLE
Added Maven-surefire-plugin to file pom.xml . Created file run_test.y…

### LINKED ISSUE #10: Implement GitHub Actions script to run tests.
## Description
As a team lead, I want GitHub action to be implemented.

The action (Pipeline) should run tests and show results.
The pipeline should be triggered on every push to GitHub, for all branches.

To make that happen, we need the [maven-surefire-plugin](https://maven.apache.org/surefire/maven-surefire-plugin/usage.html) and `YAML` config for GitHub Action.

## Additional info
Here is an example:
https://gist.github.com/oleksandr-jr/021d601c49fa46064eed98a021c8a362

https://medium.com/@alexander.volminger/ci-cd-for-java-maven-using-github-actions-d009a7cb4b8f

## Related to
#24

### PR DESCRIPTION
**Pull request by ViktorKapustianyk**
This workflow will run tests on every push to our repository.

### CODE REVIEW

**Comment by ViktorKapustianyk** [comment]
Think that everything is added correctly but even can't build the product. It shows me the issue in the interface UserMapper

**Comment by oleksandr-jr** [comment]
@ViktorKapustianyk Take a look at the latest commit here #40 and pull it

**Comment by oleksandr-jr** [comment]
@ViktorKapustianyk I just merged the fix for you. You have to pull the changes locally

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      - name: Upload test results to testhub.io
+        if: always()
+        uses: testhub-io/testhub-io.action@v0.62
+        with:
+          test_result_pattern: "target/surefire-reports/*.xml"
+          test_coverage_pattern: "target/site/jacoco/jacoco.xml"
+
+      - name: Show logs
+        run: cd logs && ls -R && echo '\n' && grep . *
+        if: always()
```
Comment (oleksandr-jr) [inline .github/workflows/run_tests.yml]: I think we can remove this.
Or fix the path and make it work :)

**Review by oleksandr-jr — CHANGES_REQUESTED** [review]

**Comment by ViktorKapustianyk** [comment]
Please check now. Hope it's ok. I didn't know on which branch I had to make changes.

**Review by oleksandr-jr — APPROVED** [review]
@Opanasenko-Mykhailo Good job!

**Pull request merged**